pom.xml

```diff
@@ -0,0 +1,499 @@
+        <version.guava>31.1-jre</version.guava>
+        <version.spring.cloud>2021.0.5</version.spring.cloud>
+        <version.spring.cloud.azure>4.5.0</version.spring.cloud.azure>
+        <version.spring-cloud-appconfiguration-config>2.10.0</version.spring-cloud-appconfiguration-config>
+        <version.spring-cloud-appconfiguration-starter>2.10.0</version.spring-cloud-appconfiguration-starter>
+        <version.spring-cloud-feature-management>2.9.0</version.spring-cloud-feature-management>
+        <version.azure-spring-boot-starter-keyvault-certificates>3.14.0</version.azure-spring-boot-starter-keyvault-certificates>
+        <version.maven-surefire-plugin>2.22.0</version.maven-surefire-plugin>
+        <mockito.version>4.5.1</mockito.version>
+      </properties>
+      <dependencyManagement>
+        <dependencies>
+          <dependency>
+            <groupId>com.azure.spring</groupId>
+            <artifactId>azure-spring-cloud-appconfiguration-config</artifactId>
+            <version>${version.spring-cloud-appconfiguration-config}</version>
+          </dependency>
+          <dependency>
+            <groupId>com.azure.spring</groupId>
+            <artifactId>azure-spring-cloud-starter-appconfiguration-config</artifactId>
+            <version>${version.spring-cloud-appconfiguration-starter}</version>
+          </dependency>
+          <dependency>
+            <groupId>com.azure.spring</groupId>
+            <artifactId>azure-spring-cloud-feature-management</artifactId>
+            <version>${version.spring-cloud-feature-management}</version>
+          </dependency>
+          <dependency>
+            <groupId>com.azure.spring</groupId>
+            <artifactId>azure-spring-cloud-feature-management-web</artifactId>
+            <version>${version.spring-cloud-feature-management}</version>
+          </dependency>
+          <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>${version.guava}</version>
+          </dependency>
+          <dependency>
+            <groupId>com.azure.spring</groupId>
+            <artifactId>azure-spring-boot-starter-keyvault-certificates</artifactId>
+            <version>${version.azure-spring-boot-starter-keyvault-certificates}</version>
+          </dependency>
+        </dependencies>
+      </dependencyManagement>
+      <repositories>
+        <repository>
+          <id>spring-release</id>
+          <name>Spring release</name>
+          <url>https://repo.spring.io/release</url>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </repository>
+      </repositories>
+      <pluginRepositories>
+        <pluginRepository>
+          <id>spring-release</id>
+          <name>Spring release</name>
+          <url>https://repo.spring.io/release</url>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </pluginRepository>
```

Comment:
Why is this necessary?

Reply:
It's used for `org.springframework.experimental:spring-native`, keeping the configuration the same with the project generated by start.spring.io

![image](https://user-images.githubusercontent.com/19926623/209528384-47425877-a658-4204-85cc-738e5df6178b.png)